src/components/modal/AccordionCard.jsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from 'react'
+
+const AccordionCard = ({ item }) => {
+  const [checked, setChecked] = useState(false)
```

Comment:
It may make more sense to name this something like `expanded` instead of `checked` because the UI is an expanding text card, not a check box.